region/maridia/outer/Mt. Everest.json

```
@@ -2572,6 +3012,66 @@
       "name": "Base",
       "requires": []
     },
+    {
+      "link": [7, 3],
+      "name": "Leave Shinecharged",
+      "requires": [
+        "Gravity",
+        {"canShineCharge": {
+          "usedTiles": 12,
+          "openEnd": 2,
+          "steepUpTiles": 5,
+          "steepDownTiles": 5
+        }}
+      ],
+      "exitCondition": {
+        "leaveShinecharged": {
+          "framesRemaining": 115
+        }
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": [
```

```suggestion
        {"types": ["super", "powerbomb"], "requires": []},
        {"types": ["missiles"], "requires": [
```
Open it from 2, then speedy jump up